### PR TITLE
GitHub collaboration - add section on merging

### PR DESCRIPTION
**Pull request by verku**

### CODE REVIEW

**Review by RedondoMA — APPROVED** [review]

**Comment by mahesh-panchal** [comment]
PR is to the wrong branch. Click edit at the top to change the branch.

File: github/gh-collaboration.qmd [hunk 1]
```diff
@@ -24,3 +24,7 @@
 ## Making changes
 
 ## Merging
+
+- Once the changes are approved by the reviewer(s), and there are no conflicts of your branch with the base branch, you can merge the changes you've made on your branch into the main branch of the target repository.
+- To do that, you click on "Pull requests" in the header line of the target repository, and select your pull request. Here you click on "Merge pull request" to perform the merging.
+- A github workflow is planned that will automatically publish the updated text on the website.
```
Comment (mahesh-panchal) [inline github/gh-collaboration.qmd:30]: I think this can be left out.

**Review by mahesh-panchal — COMMENTED** [review]
I didn't explain this clearly enough.
There is a sequence diagram on the first page that explains this process process more clearly I hope. 
See also https://github.com/NBISweden/Training-Tech-shorts/discussions/30

It would be good to describe more clearly that the teacher does the merging after they've made their review.

If you feel that this would be better served by changing the section headings please do that as I think Per's layout is a good one.

**Comment by verku** [comment]
Per's overview is great, this was indeed a misunderstanding! I've only updated my little section according to his bulletpoints, I hope this was OK.

File: github/gh-collaboration.qmd [hunk 2]
```diff
@@ -23,4 +23,6 @@
 
 ## Making changes
 
-## Merging
+## Teacher merges the pull request
+
+- Once both teacher and learner are satisfied with the updates, the teacher merges the pull request.
```
Comment (mahesh-panchal) [inline github/gh-collaboration.qmd:28]: One thing to clarify is that the teacher has a few things to merge.
- The learner's PR into their lesson branch
- And once everything is received, the lesson branch into the main branch

**Review by mahesh-panchal — APPROVED** [review]

**Comment by mahesh-panchal** [comment]
Thank you

**Pull request merged**